tools/pipelines/templates/build-npm-package.yml

```diff
@@ -218,7 +278,6 @@ stages:
             command: 'custom'
             customCommand: 'install --global npm@^10'
             customRegistry: 'useNpmrc'
-
```

Comment:
Nit: I'd say let's restore the empty lines. Some were deleted, some weren't, so it could make task grouping feel off.